external/stdk/README.md

```diff
@@ -0,0 +1,48 @@
+# Introduction
```

Comment:
```suggestion
# SmartThings Device SDK
```

I think using the name of module seems better.
How do you think?

Reply:
that seems more in line with usual rule , updated, thank you!